aten/src/ATen/native/cpu/DistanceOpsKernel.cpp

```diff
@@ -148,6 +148,54 @@ struct PDist {
     }
   }
 
+  template <typename F>
+  static void run_cdist_parallel(Tensor& result, const Tensor& t1, const Tensor& t2, const scalar_t p) {
+      const scalar_t * const t1_start = t1.data<scalar_t>();
```

Comment:
2-spaces instead of 4-spaces throughout this file.

Reply:
Fixed